### PR TITLE
Run code formatter on profile.eprof.ex

### PR DESCRIPTION
**Pull request by sublimecoder**

### CODE REVIEW

File: lib/mix/lib/mix/tasks/profile.eprof.ex [hunk 1]
```diff
@@ -95,29 +95,53 @@ defmodule Mix.Tasks.Profile.Eprof do
...
+    elixir_version_check: :boolean,
+    parallel_require: :keep
+  ]
 
   def run(args) do
     {opts, head} =
-      OptionParser.parse_head!(args,
+      OptionParser.parse_head!(
+        args,
         aliases: [r: :require, p: :parallel, e: :eval, c: :config],
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/profile.eprof.ex]: Can you please move the aliases to a module attribute as well (similar to `@switches`) so everything fits a single line? Thanks!

File: lib/mix/lib/mix/tasks/profile.eprof.ex [hunk 2]
```diff
@@ -190,7 +215,8 @@ defmodule Mix.Tasks.Profile.Eprof do
 
   defp add_totals(call_results) do
     {function_count, call_count, total_time} =
-      Enum.reduce(call_results, {0, 0, 0}, fn({_, {count, time}}, {function_count, call_count, total_time}) ->
+      Enum.reduce(call_results, {0, 0, 0}, fn {_, {count, time}},
+                                              {function_count, call_count, total_time} ->
         {function_count + 1, call_count + count, total_time + time}
       end)
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/profile.eprof.ex]: What about this:

```
Enum.reduce(call_results, {0, 0, 0}, fn {_, {count, time}}, acc ->
  {function_count, call_count, total_time} = acc
```

File: lib/mix/lib/mix/tasks/profile.eprof.ex [hunk 3]
```diff
@@ -229,7 +255,13 @@ defmodule Mix.Tasks.Profile.Eprof do
   defp format_total(total_time, total_count) do
     time_per_call = :erlang.float_to_binary(divide(total_time, total_count), [{:decimals, 2}])
 
-    ["Total", Integer.to_string(total_count), "100.00", Integer.to_string(total_time), time_per_call]
+    [
+      "Total",
+      Integer.to_string(total_count),
```
Comment (josevalim) [inline lib/mix/lib/mix/tasks/profile.eprof.ex]: Let's move those string conversions to the previous line so everything fits a single line here too.

**Comment by josevalim** [comment]
Please ignore the build failures, it was our bad.

File: lib/mix/lib/mix/tasks/profile.eprof.ex (hunk 1 above)
Comment (sublimecoder) [reply lib/mix/lib/mix/tasks/profile.eprof.ex]: So something like this?
```elixir
  @aliases [
    r: :require,·
    p: :parallel,·
    e: :eval,·
    c: :config
    ]
  
   def run(args) do
      {opts, head} =  OptionParser.parse_head!(args, aliases: @aliases, strict: @switches)
   
        Mix.Tasks.Run.run(
          ["--no-mix-exs" | args],
          opts,
          head,
          &profile_code(&1, opts),
          &profile_code(File.read!(&1), opts)
        )
   end
```

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**